incubator/hnc/test/e2e/issues_test.go

```diff
@@ -150,7 +150,7 @@ var _ = Describe("Issues", func() {
 		RunShouldContain("ActivitiesHalted (AncestorHaltActivities)", defTimeout, "kubectl hns describe", nsSubChild)
 	})
 
-	It("Should have ParentMissing condition when parent namespace is deleted - issue #716", func() {
+	It("Should have ParentMissing condition when root namespace is deleted - issue #716", func() {
```

Comment:
Why this change?

Reply:
I _feel_ like the parent namespace kind of refers to the parent-subnamespace scenario. Is there a way to differentiate it with the parent-child(not subnamespace) case?